src/renderer/components/subscription-settings/subscription-settings.vue

```diff
@@ -14,6 +14,12 @@
         :tooltip="$t('Tooltips.Subscription Settings.Fetch Feeds from RSS')"
         @change="updateUseRssFeeds"
       />
+      <ft-toggle-switch
```

Comment:
I think you should add a switch column grid and two switch columns so it matches the layout of other setting components (parental control settings, distraction free settings, privacy settings, player settings)

Reply:
I've added one now, It does look nicer. 
![2022-09-29_T18:49:07](https://user-images.githubusercontent.com/66974576/193092099-a0e5761d-66de-41a3-a85e-1074e5e934d7.png)
Should I change their order?